### ISSUE TITLE
Is null not equal to null?

### ISSUE THREAD

**Issue by mg** [bruderstein/unexpected-react-shallow#11]
My element is rendering null, which is expected. Unexpected is failing it:

```
     UnexpectedError:
expected null to have rendered null
  No matching assertion, did you mean:
```

Am I doing something wrong or is null === null a special case that is not handled?

cheers,
mg

**Comment by bruderstein** [comment]
You're right, it's not handled when the `getRenderOutput()` returns null.

I'm fairly sure we can add this without any issues for the renderer - for the output, you can just use `'to be null'`, but we can probably add it anyway.
